apidef/oas/oas.go

```diff
@@ -46,6 +51,7 @@ func (s *OAS) MarshalJSON() ([]byte, error) {
 	})
 }
 
+// Fill *OAS definition from apidef.APIDefinition
```

Comment:
Should be sentece:

```
// Fill fills .....
```

Reply:
Applied consistency to Fill() and ExtractTo() comments.